jdk_8_maven/cs/rest-gui/ocvn/pom.xml

```diff
@@ -84,6 +84,11 @@
                 <enabled>true</enabled>
             </snapshots>
         </repository>
+        <repository>
+            <id>maven.restlet.org</id>
+            <name>maven.restlet.org</name>
+            <url>https://maven.restlet.talend.com</url>
+        </repository>
```

Comment:
why modifying this?

Reply:
Hi Andrea,

Thanks for your comment. The mvn command failed to fetch ocvn in my machine (MacOS). After searching online, someone suggested that repository so I added that and it worked.

Reply:
@onurd86  we cannot have dependencies in unreliable external repositories that is not Maven Central nor GitHub. which library gave problems? note that in OCVN we had a similar issue in the past. This was solved with local jar, manually added. see `jdk_8_maven/cs/rest-gui/ocvn/persistence/lib/dozer-hibernate-model-0.3.5-06032016.jar` and the use of 
```
 <dependency>
            <groupId>nl.dries.wicket.hibernate</groupId>
            <artifactId>dozer-hibernate-model</artifactId>
            <version>${dozer.hibernate.model.version}</version>
            <scope>system</scope>
            <systemPath>${project.basedir}/lib/dozer-hibernate-model-0.3.5-06032016.jar</systemPath>
        </dependency>
```
in `jdk_8_maven/cs/rest-gui/ocvn/persistence/pom.xml`

Reply:
Hi Andrea,

In that case, I can remove that one.

Reply:
I saw that the jar is there but it did not work for my case when I removed the repository.